test/compact-range-test.js

```diff
@@ -35,3 +35,7 @@ test('test compactRange() frees disk space after key deletion', function (t) {
     });
   });
 });
+
+test('tearDown', function (t) {
+  db.close(testCommon.tearDown.bind(null, t));
+});
```

Comment:
This commit can be cherry-picked if needed.

Reply:
Yes semi colons. Added them because the rest of the file has them. Will let `standard` take care of this later, once we add it.